.yamato/compressed-sensor-test.yml

```diff
@@ -5,7 +5,7 @@ test_compressed_obs_{{ editor.version }}:
   name: Test Compressed Sensor Observation {{ editor.version }}
   agent:
     type: Unity::VM::osx
-    image: ml-agents/ml-agents-bokken-mac:0.1.4-492264
```

Comment:
Not sure this will help much, but might be faster job startup times if images are cached.

Reply:
NM, running on linux now.